govcd/sample_govcd_test_config.yaml

```diff
@@ -127,6 +127,13 @@ vcd:
       # The minimum size is 1048576 (1 MB). While theoretically smaller amounts are allowed,
       # there is an issue when using < 1MB during updates
       sizeForUpdate: 1048576
+nsxt:
+    # NSX-T manager to be used
+    manager: nsxManager1
+    # NSX-T tier-0 router used for external network
+    tier0router: tier-0-router
+    # NSX-T tier-0 VRF router used for external network
```

Comment:
For those new to NSX-T would be nice to have a comment explaining how VRF router differs from the normal one.

Reply:
It ties to down to VRF ([virtual routing and forwarding](https://en.wikipedia.org/wiki/Virtual_routing_and_forwarding)). I can't say yet I am an expert myself (at least yet) :) As I understand so far it allows to build virtual routing tables on the same Tier0 router, but I am yet to experience it better while building NSX-T from scratch.

I do know that from VCD standpoint these are differentiated and API requires different values to be passed so it makes sense to have explicit test for VRF Tier0 router.